<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Seberm</author>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>369</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sAlarm Scheduler</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../icons.qrc">
    <normaloff>:/icons/alarmIcon</normaloff>:/icons/alarmIcon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="Scheduler" name="scheduler"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>369</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Main">
    <property name="title">
     <string>&amp;Main</string>
    </property>
    <addaction name="actionNewSchedule"/>
    <addaction name="actionPreferences"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="actionReportBug"/>
    <addaction name="separator"/>
    <addaction name="actionAboutQt"/>
    <addaction name="actionAboutThisApplication"/>
   </widget>
   <widget class="QMenu" name="menuSchedule">
    <property name="title">
     <string>&amp;Schedules</string>
    </property>
    <addaction name="actionOpenSchedulesList"/>
   </widget>
   <addaction name="menu_Main"/>
   <addaction name="menuSchedule"/>
   <addaction name="menu_Help"/>
  </widget>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/closeIcon</normaloff>:/icons/closeIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionReportBug">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/bugIcon</normaloff>:/icons/bugIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;Report bug</string>
   </property>
  </action>
  <action name="actionAboutQt">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/qtIcon</normaloff>:/icons/qtIcon</iconset>
   </property>
   <property name="text">
    <string>About &amp;Qt</string>
   </property>
  </action>
  <action name="actionAboutThisApplication">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/infoIcon</normaloff>:/icons/infoIcon</iconset>
   </property>
   <property name="text">
    <string>About this application</string>
   </property>
  </action>
  <action name="actionPreferences">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/optionsIcon</normaloff>:/icons/optionsIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;Preferences</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+P</string>
   </property>
  </action>
  <action name="actionNewSchedule">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/loadIcon</normaloff>:/icons/loadIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;New schedule</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionScheduleEdit">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/editIcon</normaloff>:/icons/editIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;Edit</string>
   </property>
  </action>
  <action name="actionScheduleRemove">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/deleteIcon</normaloff>:/icons/deleteIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;Remove</string>
   </property>
  </action>
  <action name="actionOpenSchedulesList">
   <property name="icon">
    <iconset resource="../../icons.qrc">
     <normaloff>:/icons/openIcon</normaloff>:/icons/openIcon</iconset>
   </property>
   <property name="text">
    <string>&amp;Open schedules list</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Scheduler</class>
   <extends>QListView</extends>
   <header>scheduler.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../icons.qrc"/>
 </resources>
 <connections/>
</ui>
